### PR TITLE
Fix apply for non default terraform binary path

### PR DESCRIPTION
**Pull request by apielasa**
Terraform version needs to be verified when the path for terraform binary file is set.

### CODE REVIEW

**Review by arm4b — CHANGES_REQUESTED** [review]
Thanks for the contribution! 

Could you please also add a Changelog record and bump a patch version in https://github.com/StackStorm-Exchange/stackstorm-terraform/blob/master/pack.yaml#L8 ?

**Comment by apielasa** [comment]
> Thanks for the contribution!
> 
> Could you please also add a Changelog record and bump a patch version in https://github.com/StackStorm-Exchange/stackstorm-terraform/blob/master/pack.yaml#L8 ?

Sure, no problem.
I have just updated the PATCH version

File: pack.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ runner_type: "python-script"
 description: Terraform integrations
 keywords:
   - terraform
-version: 2.0.0
+version: 2.0.1
```
Comment (arm4b) [inline pack.yaml:8]: Needs a Changelog record for 2.0.1 here:
https://github.com/StackStorm-Exchange/stackstorm-terraform/blob/master/CHANGES.md

**Comment by armab** [comment]
Please also don't forget to sign the CLA above https://github.com/StackStorm-Exchange/stackstorm-terraform/pull/39#issuecomment-1327472231 ^^.

File: pack.yaml (hunk 1 above)
Comment (apielasa) [reply pack.yaml:8]: @armab  - changelog updated.

**Comment by arm4b** [comment]
All good, @apielasa just missing a CLA from your side (see instructions https://github.com/StackStorm-Exchange/stackstorm-terraform/pull/39#issuecomment-1327472231) to merge it.

**Comment by apielasa** [comment]
CLA signed.

**Review by arm4b — APPROVED** [review]
Thanks!

**Pull request merged**